src/gettingstarted/local-development/connect-services.md

```diff
@@ -0,0 +1,52 @@
+# Local development
+
+## Connect to services
+
+> **Note:** If your application does not contain any services, you do not need to open a tunnel and can proceed to the next step.
+
+Now that you have a local copy of your application code, you can make changes to the project without pushing to Platform.sh each time to test them. Instead you can locally build your application using the CLI, even when its functionality depends on a number of services.
+
+<asciinema-player src="/videos/asciinema/tunnel-open.cast" preload=1 autoplay=1 loop=1></asciinema-player>
+
+1. **Open an SSH tunnel to connect to your services**
+
+    Open local [SSH tunnels](/development/local/tethered.md#ssh-tunneling) to your environment's services.
+
+    ```bash
+    platform tunnel:open
+    ```
+
+2. **Export environment variables**
+
+    Platform.sh utilizes environment variables called Relationships within the application container. These store the credentials needed to connect to individual services. In order to connect with them remotely using the SSH tunnel you will need to mimic the same environment variables locally.
+
+    ```bash
+    export PLATFORM_RELATIONSHIPS="$(platform tunnel:info --encode)"
+    ```
+    In order to use these credentials to connect to your services, it is also necessary that those services are locally installed.
```

Comment:
This... is not exactly true.  The clients for those services are needed, but may be built into your application.  You do not need Solr installed locally in order to use a tethered configuration; the point of tethering is so that you don't need to install Solr locally.  This line is misleading.